inst/example_workflow/2_metrics/kri0001.yaml

```
@@ -0,0 +1,72 @@
+meta:
+  Type: Analysis
+  ID: kri0001
+  GroupLevel: Site
+  Abbreviation: AE
+  Metric: Adverse Event Rate
+  Numerator: Adverse Events
+  Denominator: Days on Study
+  Model: Normal Approximation
+  Score: Adjusted Z-Score
+  AnalysisType: rate
+  Threshold: -2,-1,2,3
+  AccrualThreshold: 30
+  AccrualMetric: Denominator
+spec:
+  Mapped_AE:
+    subjid:
+      type: character
+  Mapped_SUBJ:
+    subjid:
+      type: character
+    invid:
+      type: character
+    timeonstudy:
+      type: integer
+steps:
+  - output: vThreshold
+    name: ParseThreshold
```

```suggestion
    name: gsm.core::ParseThreshold
```

does rest of file need appropriate namespacing too?